requirements.txt

```diff
@@ -54,8 +54,7 @@ greenlet==3.1.0
     # via sqlalchemy
 gspread==6.1.2
     # via hdx-python-scraper
-hapi-schema==0.8.17
-    # via hapi-pipelines (pyproject.toml)
+hapi-schema@git+https://github.com/OCHA-DAP/hapi-sqlalchemy-schema@develop
```

Comment:
Is this the trick to get it to work with GitHub Actions? I'll need to remember this!

Reply:
This plus the change in pyproject.toml - I'm actually not sure which one is key so I do both!